feature/session/src/main/java/io/github/droidkaigi/confsched2020/session/ui/SessionsFragment.kt

```diff
@@ -98,8 +101,12 @@ class SessionsFragment : DaggerFragment() {
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
         val initialPeekHeight = sessionSheetBehavior.peekHeight
+        val gestureNavigationBottomSpace =
+            if (isEdgeToEdgeEnabled()) (48 * resources.displayMetrics.density).toInt() else 0
```

Comment:
Can you use a dimens resource?

Reply:
Yes, that is better. Modify to get from dimen resource.